lib/mysqlDao.js

```diff
@@ -356,6 +357,129 @@ class MysqlDao {
     }
 
     /**
+     * @param {object} whereClause
+     * @param {MysqlDaoGetAllOptions}
+     * @param {ReadableOptions} [streamOptions]
```

Comment:
Eh seems like you can leave it if WebStorm figures it out with the typescript library. @typedef is probably overkill